### PR TITLE
Add tree view status label hook, fixes #760

### LINKED ISSUE #760: Enhance page tree view with mirroring information
**Is your feature request related to a problem? Please describe.**
Sitemanager doesnt have the possibility to see which site is mirrored and from where the content is mirrored in the overview and on the single site this should be displayed more present.

**Describe the solution you'd like**
Nice implementation would be a prominent different colour, which is explained with a legend, and the information from where the site is mirrored with titel or/and site-id, this should be clickable. 

On the mirrored site there should be also a additional information, from where the content is mirrored and that the content is mirrored.

**Describe alternatives you've considered**
None, but you are free to design as you want... but i also requested a ux assist

### PR DESCRIPTION
**Pull request by svenseeberg**
#### Short description of what this resolves:
This plugin introduces a hook into the tree view plugin that allows attaching labels to pages.

image

#### Changes proposed in this pull request:
- Plugin adjustment to call new function in cms-tree-view-extension
- Add hook that is used by ig-attach-content and ig-revisions

**Fixes**: #760

### Your checklist for this pull request

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [x] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *develop*.
- [x] Check the commit's or even all commits' message styles matches your requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

### CODE REVIEW

**Comment by svenseeberg** [comment]
Translations are still missing.

File: wp-activate-plugins.sh [hunk 1]
```diff
@@ -8,9 +8,8 @@ cms-tree-page-view \
 disable-new-user-notifications \
 events-manager \
 firebase-notifications \
+ig-attach-content \
 ig-cms-tree-page-view-extensions \
-ig-content-loader-base \
-ig-content-loader-instance \
```
Comment (timoludwig) [inline wp-activate-plugins.sh]: were these lines removed unintentionally?

File: wp-content/plugins/ig-cms-tree-page-view-extensions/cms-tree-page-view-extensions.php [hunk 2]
```diff
@@ -16,3 +16,21 @@ function wpse_73006_submenu_order( $menu_ord )
...
+/**
+ * This function adds a filter ig-cms-tree-view-status that allows attaching
+ * labels to the page tree view.
+ *
+ * @param integer $post_id ID of the post item in the tree view
+ * @param array $post_status contains a list of labels (string)
+ * @return string
+ */
+function ig_tree_view_labels ( $post_id, $post_status ) {
+    $post_status = ( count(array($post_status)) > 0 && $post_status != 'publish' ? array($post_status) : array() );
```
Comment (timoludwig) [inline wp-content/plugins/ig-cms-tree-page-view-extensions/cms-tree-page-view-extensions.php]: The condition `count(array($post_status)) > 0` will always be true, even if `$post_status` is an empty string or `null`.

Comment (svenseeberg) [reply wp-content/plugins/ig-cms-tree-page-view-extensions/cms-tree-page-view-extensions.php]: Thanks for capturing that. The array() function does not belong there.

Comment (svenseeberg) [reply wp-content/plugins/ig-cms-tree-page-view-extensions/cms-tree-page-view-extensions.php]: Oh yes, strlen() is what I need here.

**Pull request merged**